src/pages/index.js

```diff
@@ -188,8 +187,197 @@ const App = ({ data }) => (
+            expected.
+          </em>
+        </p>
+        <h3>August 2009</h3>
+        <p>
+          Kyle texts Shannon to ask her on a date and she doesn't respond for
+          several weeks because her phone is broken 😅
+        </p>
+        <p>They plan to go on another date, and then this happens:</p>
+        <p>
+          <strong>Shannon:</strong>
+          {` `}
+          “Kyle, I have to reschedule our date. I’m in the hospital with
+          appendicitis and go into surgery in 45 minutes.”
+        </p>
+        <p>
+          <strong>Kyle:</strong> “That is the best excuse I’ve ever heard for
+          rescheduling a date! Good luck!”
+        </p>
+        <p>
+          {" "}
+          They finally go on date a few days after Shannon has her appendix out.
+          Shannon was probably on pain killers. Neither of them remember the
+          events of the date.
+        </p>
+        <h3>October 2009</h3>
+        <p>Shannon exits to southern California for a 1.5 year LDS mission.</p>
+        <h3>January 2011</h3>
+        <p>Kyle exits to northern California to chase startup dreams.</p>
+        <h3>May 2015</h3>
+        <p>Shannon messages Kyle on Spotify</p>
+        <p>
+          It’s been 6 years since we last talked and then Shannon follows Kyle
+          on Spotify and thinks this song he's listening to is cool.
+        </p>
+        <iframe
+          src="https://open.spotify.com/embed/track/01txDiOnBPfMrVQwPQtDHR"
+          width="300"
+          height="380"
+          frameborder="0"
+          allowtransparency="true"
+          allow="encrypted-media"
+        />
+        <p>
+          Through Facebook, she realizes that Kyle lives in San Francisco and
+          since she’s already planning on road tripping through California with
+          a friend, she messages him on Facebook to see if he wants to hang out.
+        </p>
+        <p>
+          <strong>Shannon:</strong> Hey me and a friend are going to be in Palo
+          Alto/San Fran over 4th of July weekend! It'd be cool to meet up
+          sometime! And any suggestions for cool 4th o' July things...?
+        </p>
+        <p>
+          <strong>Kyle:</strong>
+          {` `}
+          Would love to hang out plus will brainstorm some ideas … [later] sooo
+          I had an idea for something to do while you and your friend are here.
+          A friend and I were talking about this who also has some friends
+          coming into town. Basically Saturday the fourth we'd do a "bakery
+          crawl" starting at my house in inner sunset and visiting first a
+          bakery near me and then city hiking our way to a bakery in the
+          mission. It's a pretty walk and you'd get to eat lots of delicious
+          bread.
+        </p>
+        <h3>July 4, 2015</h3>
+        <Img fluid={data.sf.childImageSharp.fluid} />
+        <p>
+          <em>post-Thai food happiness w/ Liz Lyon</em>
```

Comment:
Just to make it clear who's who, "post-Thai food happiness with Liz Lyon, Shannon Soper, and Kyle Mathews"